tests/phpunit/tests/template.php

```diff
@@ -502,21 +530,46 @@ public function test_locate_template_uses_current_theme() {
 	}
 
 	/**
-	 * Tests that wp_start_template_enhancement_output_buffer() does not start a buffer when no filters are present.
+	 * Tests that wp_start_template_enhancement_output_buffer() does not start a buffer in a block theme when no filters are present.
 	 *
 	 * @ticket 43258
+	 * @ticket 64099
 	 * @covers ::wp_should_output_buffer_template_for_enhancement
 	 * @covers ::wp_start_template_enhancement_output_buffer
+	 * @covers ::wp_load_classic_theme_block_styles_on_demand
 	 */
-	public function test_wp_start_template_enhancement_output_buffer_without_filters_and_no_override(): void {
-		remove_all_filters( 'wp_template_enhancement_output_buffer' );
+	public function test_wp_start_template_enhancement_output_buffer_without_filters_and_no_override_in_block_theme(): void {
+		switch_theme( 'block-theme' );
+		wp_load_classic_theme_block_styles_on_demand();
+
 		$level = ob_get_level();
 		$this->assertFalse( wp_should_output_buffer_template_for_enhancement(), 'Expected wp_should_output_buffer_template_for_enhancement() to return false when there are no wp_template_enhancement_output_buffer filters added.' );
 		$this->assertFalse( wp_start_template_enhancement_output_buffer(), 'Expected wp_start_template_enhancement_output_buffer() to return false because the output buffer should not be started.' );
 		$this->assertSame( 0, did_action( 'wp_template_enhancement_output_buffer_started' ), 'Expected the wp_template_enhancement_output_buffer_started action to not have fired.' );
 		$this->assertSame( $level, ob_get_level(), 'Expected the initial output buffer level to be unchanged.' );
 	}
 
+	/**
+	 * Tests that wp_start_template_enhancement_output_buffer() does start a buffer in classic theme.
+	 *
+	 * @ticket 43258
+	 * @ticket 64099
+	 * @covers ::wp_should_output_buffer_template_for_enhancement
```

Comment:
This applies in a few places, an empty line between the ticket and covers annotations.

```suggestion
	 * @ticket 64099
	 *
	 * @covers ::wp_should_output_buffer_template_for_enhancement
```